src/runtime/HalidePyTorchHelpers.h

```diff
@@ -90,16 +91,29 @@ inline Buffer<scalar_t> wrap(at::Tensor &tensor) {
 #endif
     Buffer<scalar_t> buffer;
```

Comment:
...why not just

    return Buffer<scalar_t>(pData, dims);

?

Reply:
fair!